storm/src/main/storm/mesos/NimbusMesosScheduler.java

```diff
@@ -17,13 +17,7 @@
  */
 package storm.mesos;
 
-import org.apache.mesos.Protos.ExecutorID;
-import org.apache.mesos.Protos.FrameworkID;
-import org.apache.mesos.Protos.MasterInfo;
-import org.apache.mesos.Protos.Offer;
-import org.apache.mesos.Protos.OfferID;
-import org.apache.mesos.Protos.SlaveID;
-import org.apache.mesos.Protos.TaskStatus;
+import org.apache.mesos.Protos.*;
```

Comment:
Can you revert the * import. We don't like those.

Reply:
Sure, I didn't realize my IntelliJ settings had that enabled by default.